### DISCUSSION TITLE
Why did this script file `test/scripts/webhook-create-signed-cert.sh` disappear after version 1.6?

### DISCUSSION THREAD

**Discussion by zeaone** [rook/rook#11237] [Q&A]
Why did this script file `test/scripts/webhook-create-signed-cert.sh` disappear after version 1.6? Did the location change?Or has the usage changed?

**Comment by subhamkrai** [comment]
@zeaone you want to start the webhook for rook? or what is the requirement?

earlier we have to run some script to get the webhook running, now we use cert-manager for the webhooks you just need to have cert-manager pre-exists in the cluster.

**Reply by parth-gr** [reply]
@subhamkrai they might be looking for the upgrade steps

**Reply by zeaone** [reply]
yes, i want use webhook in rook.In order to prevent user's misoperation and prevent cephcluter from being deleted by mistake, causing the cluster to crash. My current rook is version 1.4. How to use `cert-manager` to enable webhook? tanks

**Reply by subhamkrai** [reply]
> yes, i want use webhook in rook.In order to prevent user's misoperation and prevent cephcluter from being deleted by mistake, causing the cluster to crash. My current rook is version 1.4. How to use `cert-manager` to enable webhook? tanks

you can use the script to install cert-manager: https://github.com/rook/rook/blob/master/tests/scripts/deploy_cert_manager.sh or the official website and the install restart the rook operator.

Let me know how that goes.

**Reply by zeaone** [reply]
OK, I'll try this way and give you feedback

**Reply by zeaone** [reply]
Why is my `admission-controller pod`not aware of CephCluster changes? Is it because there is something wrong with my path configuration? If yes, how should it be configured? My overall configuration is as foll.  my rook version is **v1.4**

1.the yaml for ValidatingWebhookConfiguration
![image](https://user-images.githubusercontent.com/15962991/198828317-192d6e2a-3d1d-42be-80ff-409e90174929.png)

2.the yaml for csr 
![image](https://user-images.githubusercontent.com/15962991/198830209-ee114d2e-15ac-44de-9f40-9db1bd14f503.png)
![image](https://user-images.githubusercontent.com/15962991/198830229-c3dd167b-e5d7-4b19-a717-9acd21935122.png)

3.my rook version is **v1.4**

**Reply by subhamkrai** [reply]
the current approach is with latest rook version, I think from rook ~1.8. so you need to upgrade before upgrading make sure webhook disabled.
